packages/odyssey-react/src/components/Banner/Banner.stories.tsx

```diff
@@ -12,9 +12,9 @@
 
 import { Story } from "@storybook/react";
 import { useArgs } from "@storybook/client-api";
-import Banner from ".";
+import { Banner } from ".";
 import Link from "../Link";
-import type { Props } from ".";
+import type { BannerProps } from ".";
```

Comment:
named prop types conflict in our barrel files unless we go back to a more specific naming scheme for exports